packages/magika.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Continue'
```

Comment:
The only reason we have `$ErrorActionPreference = 'Continue'` in libraries.python3.vm is to install other libraries if one fails. There is only one library here, so we want to ensure it installs without errors. Which error are you exactly getting? (If you change this to make the linter happy we should be able to see it in the run)